models/marts/schema.yml

```
@@ -0,0 +1,29 @@
+version: 2
+models:
+  - name: customers
+    columns:
+      - name: customer_id
+        tests:
+          - unique
+          - not_null
+  - name: dim_customers
+    columns: 
+      - name: customer_id
+        tests:
+          - unique
+          - not null
+      - name: first_name
+        tests:
+          - relationships:
+              to: ref('stg_customers')
+              field: first_name
+      - name: last_name
+        tests:
+          - relationships:
+              to: ref('stg_customers')
+              field: last_name
+  - name: fct_orders
+    columns:
+      - name: order_id
+        tests:
+          - not null
```

Might want to add a unique test to this as well?
```suggestion
          - not null
          - unique
```